### ISSUE TITLE
Don't emit Object type

### ISSUE THREAD

**Issue by bradleyayers** [bradleyayers/getdocs2ts#2]
The TSLint rules in DefinitelyTyped prohibit Object:

```
ban-types   Don't use 'Object' as a type. Avoid using the `Object` type. Did you mean `object`?
```

**Comment by timjb** [comment]
This is fixed in https://github.com/timjb/buildtypedefs/commit/3f6e336d1e9bd78663ff5665a4c2e742f5a2726a

**Comment by bradleyayers** [comment]
I don't think we want to replace `Object` with `object`, but rather the indexed type `{ [key: string]: any }`.

Example: http://www.typescriptlang.org/play/#src=const%20attrs%3A%20object%20%3D%20%7B%20myAttr%3A%20''%20%7D%3B%0D%0Aattrs.myAttr%3B

**Comment by bradleyayers** [comment]
So actually `object` is fine for the type of parameters, but for return values I think we want to go with `{ [key: string]: any }` else it's not possible to access properties.
